documentation/modules/exploit/linux/local/polkit_dbus_auth_bypass.md

```diff
@@ -92,7 +92,7 @@ msf6 exploit(linux/local/polkit_dbus_auth_bypass) > run
 [*] Attempting to create user msf
 [+] User msf created with UID 1019
 [*] Attempting to set the password of the newly create user, msf, to: NpJsQSti
-[+] Obtained code execution has root!
```

Comment:
haz root*

Reply:
Think this should be solved with https://github.com/rapid7/metasploit-framework/pull/15368/commits/5011e950d30510d3fdaa5212bf513bf109f72d80?